Eatery/Extensions/Eatery+Extension.swift

```diff
@@ -0,0 +1,176 @@
+import Foundation
+import SwiftyJSON
+import UIKit
+
+enum EateryStatus {
+    case open(String, String)
+    case closing(String, String)
+    case closed(String, String)
+}
+
+private let ShortDateFormatter: DateFormatter = {
+    let formatter = DateFormatter()
+    formatter.dateFormat = "h:mma"
+    return formatter
+}()
+private let kEateryAppendix = JSON(try! Data(contentsOf: Bundle.main.url(forResource: "appendix", withExtension: "json")!)).dictionaryValue
```

Comment:
Rename to `eateryAppendix`

Reply:
I am going to leave this for now. This was in the previous versions, but I had to add it back for the watch app. Will eventually be removed